noir/noir-repo/noir_stdlib/src/lib.nr

```diff
@@ -30,6 +30,13 @@ mod append;
 mod mem;
 mod panic;
 
+// Keep following constants in sync with the enum acir_format::PROOF_TYPE in recursion_constraint.hpp
```

Comment:
These constants should be maintained by the user of certain backend, which in this case is aztec-packages. We still have `verify_proof` which hardcodes the plonk proof type as to be non-breaking for UP users, but eventually we will have just `verify_proof_with_type` in the noir stdlib.

Reply:
Ok, I will move them to elsewhere probably in constants.nr